package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ethos-design-system",
-  "version": "1.1.211",
+  "version": "1.1.213",
```

Comment:
why do we need to update EDS for this change?

Reply:
Basic style of the nora button comes from EDS. We could override it Nora styles but seems like this may be a better place. I may be wrong though. Not sure who is the best person to ask.

Reply:
Or is the question why we need to bump the version?